app.py

```diff
@@ -72,10 +80,14 @@ def get_full_dirname(dirname, do_overwrite):
 
     if os.path.exists(full_path_dir):
         if not os.path.isdir(full_path_dir):
-            click.echo('\n{} exists, and is not a directory!\n'.format(full_path_dir))
+            click.echo('\n{} exists, and is not a directory!\n'.format(
+                full_path_dir)
+            )
             exit(1)
         if os.listdir(full_path_dir) and not do_overwrite:
-            if not click.confirm('\nDirectory "{}" exists. Overwrite?'.format(full_path_dir)):
+            if not click.confirm('\nDirectory "{}" exists. Overwrite?'.format(
+                    full_path_dir)
+            ):
                 sys.exit(1)
```

Comment:
One longer line is much  better here.

Reply:
Is it okay that it is violating PEP8? 
If yes, I can revert this.

Reply:
We can only violate a rule that is enforced :) The purpose of PEP8 is to improve readability, not to cause unnecessary eye strain. [Here](https://www.youtube.com/watch?v=wf-BqAjZb8M) is an excellent talk given by [Raymond Hettinger](https://twitter.com/raymondh?lang=en) on this topic.

Reply:
I've got your point and I some kind of agree, except that in most project I was or working it won't be allowed by CI/CD to pass checks. :)